MODULE.bazel

```diff
@@ -0,0 +1,57 @@
+module(
```

Comment:
Add copyright boilerplate.

(Here and everywhere else where there's a new file.)